src/App.tsx

```diff
@@ -54,147 +56,133 @@ export default function App() {
-                setResults(detailedResultsCalculation);
-                let IDhtml = [];
-                if (detailedResultsCalculation !== null && calculationOptions.calculateHTML === true) {
-                  let i: any;
-                  for (i in detailedResultsCalculation.results[0].detailedResults) {
-                    if (detailedResultsCalculation.results[0].detailedResults !== undefined) {
-                      IDhtml.push({
-                        groupId: detailedResultsCalculation.results[0].detailedResults[i].groupId,
-                        html: detailedResultsCalculation.results[0].detailedResults[i].html!
-                      });
-                    }
-                  }
-                  setHTMLs(IDhtml);
-                } else {
-                  setHTMLs([]);
-                }
-              } else if (outputType === 'measureReports') {
-                const mrResults = Calculator.calculateMeasureReports(measureBundle, patientBundle, options);
-                const mrs = mrResults.results;
-
-                if (options.calculateHTML) {
-                  const htmls: HTML[] = mrs.map(m => ({
-                    groupId: m.id || '',
-                    html: m.text?.div || ''
-                  }));
-                  setHTMLs(htmls);
-                }
-
-                setResults(mrResults);
-              } else if (outputType === 'gapsInCare') {
-                setResults(Calculator.calculateGapsInCare(measureBundle, patientBundle, options));
-              }
-            }}
+            onClick={calculate}
+            className={classes.buttons}
+            disabled={measureFile.content === null || patientFile.content === null}
           >
             Calculate
           </Button>
         </Grid>
-        <Grid container>
-          <Grid container item xs={6} direction="row">
-            <div>
-              <h2>Results:</h2>
-              {results && (
-                <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
-              )}
-              {results && (
-                <Button
-                  variant="contained"
-                  color="primary"
-                  onClick={() => {
-                    fileDownload(
-                      JSON.stringify(results),
-                      measureFileName?.includes('.json')
-                        ? `results-${measureFileName}`
-                        : `results-${measureFileName}.json`
-                    );
-                  }}
-                >
-                  Download
-                </Button>
-              )}
-            </div>
-          </Grid>
-          <Grid container item xs={6} direction="row">
-            {htmls &&
-              htmls.map(html => {
-                return (
-                  <div key={html.groupId} className={classes.highlightedMarkup}>
-                    <h2>HTML:</h2>
```

Comment:
We have sort of a history-caused error for displaying the html...
If we do a gaps in care calculation for a numerator patient, the highlighted html will not be shown. But if we switch to denominator (html shown) and then back to numerator, it will continue showing the html, but it will be incorrect. 
My vote would be to still show the proper highlighted html for a numerator patient for a gaps in care calculation, but you could potentially make sure it's resetting the html to empty when switching back to the numerator patient.

Reply:
<img width="860" alt="Screen Shot 2021-05-26 at 11 21 57 AM" src="https://user-images.githubusercontent.com/2643955/119686743-a31ad480-be14-11eb-8bfe-1a8f9d19c6e3.png">
.
.
.
<img width="677" alt="Screen Shot 2021-05-26 at 11 22 03 AM" src="https://user-images.githubusercontent.com/2643955/119686742-a31ad480-be14-11eb-9278-ad88c7376555.png">

Reply:
This is a general comment, but I probably should have placed it at line 54 of src/components/Results/Results.tsx (this comment is linked from that line)

Reply:
This would involve changing the gaps output of fqm-execution I think unfortunately.

For gaps, the HTML is pulled off of the measure report narrative (remember that gaps doesn't return any of the detailed results, just the gaps bundle. So when we have a gaps bundle that contains a measure report, we can grab the HTML from that.

In order to display the narrative for a numerator patient, we'd need to make the gaps output for 0 gaps basically be a bearbones bundle with a measure report and no detected issues. I think that is reasonable, but should also be a separate task haha

Reply:
I do think we probably should wipe out the HTML before we display new results. Just incase they are not included in the new run.

Reply:
I guess this needn't be that complicated.

Thoughts on me just clearing HTML and results at the start of every calculate call?